### PR TITLE
Warn user for Java requirement #27

### PR DESCRIPTION
**Pull request by apupier**
unfortunately cannot reuse code, I opened a request for that https://github.com/redhat-developer/vscode-java/issues/764 
for now I duplicated the code

### CODE REVIEW

**Comment by apupier** [comment]
@lhein @bfitzpat can you review this PR please?

File: package.json [hunk 1]
```diff
@@ -62,7 +62,10 @@
 		"tslint": "gulp tslint"
 	},
 	"dependencies": {
+		"chunk": "^0.0.2",
```
[the remark this answers is not in the document]
Comment (apupier) [reply package.json]: good catch, seems I missed something during a rebase.

File: src/requirements.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
+// file copied from vscode-java, the vscode-java API is not exposed and license is EPL so possible
```
Comment (lhein) [inline src/requirements.ts]: the whole file seems to use spaces for indentation

Comment (apupier) [reply src/requirements.ts]: fixed

**Comment by lhein** [comment]
I guess the only way to test it is to uninstall java from my machine?

**Comment by apupier** [comment]
> I guess the only way to test it is to uninstall java from my machine?

yes, it is not very convenient.
it is possible to mock/modify code to return that requirements are not fulfilled and test "our part" of the code. i expect the duplicated code is tested in vscode-java codebase (and hope that they will extract it soon, or we migth need to do it as it will be nice to have it also in Camel tooling and maybe for Apicurito if a java server is provided)

**Comment by apupier** [comment]
created https://github.com/jboss-fuse/vscode-atlasmap/issues/44

**Pull request merged**